src/iop/colorbalancergb.c

```diff
@@ -646,6 +655,158 @@ void process(struct dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece, const
 }
 
 
+#ifdef OPENCL
```

Comment:
should be HAVE_OPENCL. the actual code does not activate OpenCL.

Reply:
fixed